WordPressVIPMinimum/Sniffs/Functions/RestrictedFunctionsSniff.php

```diff
@@ -98,7 +98,7 @@ public function getGroups() {
 			// @link https://docs.wpvip.com/technical-references/code-review/vip-notices/#h-switch_to_blog
 			'switch_to_blog' => [
 				'type'      => 'error',
-				'message'   => '%s() is not something you should ever need to do in a VIP theme context. Instead use an API (XML-RPC, REST) to interact with other sites if needed.',
+				'message'   => '%s() may not work as expected since it only changes the database context for the blog and does not load the plugins or theme of that site. Filters or hooks on the blog you are switching to will not run.',
```

Comment:
Can we mention that they should always call `restore_current_blog()` afterwards, since that is sometimes a cause of difficult to determine logic bugs? I would say that encouraging that, is more important than telling them how `switch_to_blog()` works.

Reply:
@GaryJones I wonder if it would be better to have a supplementary sniff to flag for a missing `restore_current_blog()` after a `switch_to_blog()` with that type of messaging recommendation?

Reply:
Sure. I think there may even be one around somewhere.

Reply:
I agree with @rebeccahum that the check for `restore_current_blog()` would probably best be done in a separate sniff (and would actually be a good one to add to WPCS). 👍